CHANGELOG.md

```diff
@@ -24,6 +24,17 @@
 - iOS: Sync metadata can be reset using the `resetBookmarksMetadata` method
   ([#1092](https://github.com/mozilla/application-services/pull/1092))
 
+## Push
+
+### Breaking Changes
+
+- `PushManager.verify_connection()` now returns a boolean. `true`
```

Comment:
I think the API we expose is verifyConnection, no?

Reply:
D'oh, yeah, I'll fix that.